### PR TITLE
Add COSY and INADEQUATE docs pages

### PR DESCRIPTION
**Pull request by mccarthy677**
The docs pages are showing up right after the first example in the simulation gallery, I think I could only move them to the end by renaming everyting to plot_01 instead of plot_1 to make them be in order, but I didn't know if that was necessary.

### CODE REVIEW

File: examples_source/2D_simulation(crystalline)/plot_10_COSY.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        sp.FFT(dim_index=(0, 1)),
+    ]
+)
+
+spectra_list = [
+    proc.apply_operations(dataset=method.simulation) for method in sim.methods
+]
+
+flipped = spectra_list[1].fft(axis=0).conj().fft(axis=0)
+flipped.dimensions[1] = spectra_list[0].dimensions[1]
```
Comment (deepanshs) [inline examples_source/2D_simulation(crystalline)/plot_10_COSY.py:109]: can you separate the two `spectra_list` simulations into `path` and `anti_path` and then show the proceesing with these variables? I think that would be clear. Based on the methods list, the zero index is `anti_path` and first is `path`

Comment (mccarthy677) [reply examples_source/2D_simulation(crystalline)/plot_10_COSY.py]: resolved in latest commit

File: examples_source/2D_simulation(crystalline)/plot_10_COSY.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    for o_sgn, t_queries in zip(offset_sign, trans_queries)
+]
+
+# %% [markdown]
+# Create the Simulator object with the spin system and two methods, and run the
+# simulation.
+
+# %%
+sim = Simulator(spin_systems=[ax_system], methods=methods)
+sim.run()
```
[the remark this answers is not in the document]
Comment (mccarthy677) [reply examples_source/2D_simulation(crystalline)/plot_10_COSY.py]: resolved in latest commit

File: examples_source/2D_simulation(crystalline)/plot_10_COSY.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        sp.apodization.TopHat(rising_edge="0 s", dim_index=0),
+        sp.apodization.TopHat(rising_edge="0 s", dim_index=1),
+        sp.apodization.Exponential(FWHM="1 Hz", dim_index=0),
+        sp.apodization.Exponential(FWHM="1 Hz", dim_index=1),
+        sp.FFT(dim_index=(0, 1)),
+    ]
+)
+
+processed_path = proc.apply_operations(dataset=sim.methods[0].simulation)
+processed_antipath = proc.apply_operations(dataset=sim.methods[1].simulation)
```
Comment (deepanshs) [inline examples_source/2D_simulation(crystalline)/plot_10_COSY.py:107]: isn't method at index 0 antipath (-1 -> -1) and at 1 is path  (+1 -> -1)?

Comment (mccarthy677) [reply examples_source/2D_simulation(crystalline)/plot_10_COSY.py:107]: No, path is  -1,-1 and antipath is +1,-1, see slide 32 at this link: https://grandinetti.org/resources/2024-NMRSequenceDesign.pdf

Comment (mccarthy677) [reply examples_source/2D_simulation(crystalline)/plot_10_COSY.py:107]: also, when I flip them, the nu_1 dimension goes negative, so it doesn't look correct in the simulation either

**Review by deepanshs — APPROVED** [review]

**Pull request merged**